app/controllers/world_wide_taxons_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WorldWideTaxonsController < ApplicationController
+  def show
+    setup_content_item_and_navigation_helpers(taxon)
+
+    taxon_template =
```

Comment:
The return values of `presented_taxon.rendering_type` are identical to the [case statement return values](https://github.com/alphagov/collections/pull/533/files#diff-e4895bed2fbf26d3e0d977f41ef5d9cfR24) aren't they?
Seems like this case statement could go away if so.

Reply:
@steventux This controller was copied wholesale from the existing taxons_controller. The only changes made were to the class names for `Taxon` and `TaxonPresenter`.
But yes, I think you're right about the case statement!

Reply:
The `:grid` case statement was also removed here.